### DISCUSSION TITLE
New versions of DBeaver lost formatting functionality

### DISCUSSION THREAD

**Discussion by LordMilutin** [dbeaver/dbeaver#20122] [Q&A] [answered]
### Description

Hi.

In the past few versions, there is no longer formatting functionality when it comes to Java SQL queries.

"select\r\n" +
"z.broj COUNTRY_ID,\r\n" +
"z.sifra PAYROLL_ID\r\n" +
"from  zaposleni z\r\n" +
"order by z.id"

Previously, this would automatically be reformatted when pasted into the DBeaver script editor.

select
z.broj COUNTRY_ID,
z.sifra PAYROLL_ID
from  zaposleni z
order by z.id

### DBeaver Version

23.0.5

### Operating System

Windows 11

**Comment by E1izabeth** [comment, marked as answer]
Please, turn on this setting.
![image](https://github.com/dbeaver/dbeaver/assets/28875055/dfa4f8f4-12c9-45e9-88ae-530b2fd05ac0)

**Comment by LordMilutin** [comment]
@E1izabeth 
Thank you so much for this!
